plugins/main/public/components/overview/vulnerabilities/searchbar/use-search-bar-configuration.test.ts

```diff
@@ -0,0 +1,220 @@
+import { renderHook } from '@testing-library/react-hooks';
+import '@testing-library/jest-dom/extend-expect';
+// osd dependencies
+import { Start, dataPluginMock } from '../../../../../../../src/plugins/data/public/mocks';
+import {
+  Filter,
+  IndexPattern,
+  Query,
+  TimeRange,
+} from '../../../../../../../src/plugins/data/public';
+import { createFilterManagerMock } from '../../../../../../../src/plugins/data/public/query/filter_manager/filter_manager.mock';
```

Comment:
Unused code
```suggestion
```

Reply:
Resolved in https://github.com/wazuh/wazuh-kibana-app/pull/5960/commits/2348731fb1410f74abdaeee6920585942d2e2c0a